contents/docs/libraries/next-js.md

```diff
@@ -127,7 +152,27 @@ export function PostHogPageview() {
         }
       )
     }
-  }, [pathname, searchParams])
+  }, [pathname, searchParams, posthog])
+  
+  return null
+}
+```
+
+</MultiLanguage>
+
+
+<MultiLanguage>
+
+```js
+// app/providers.js
+'use client'
+import posthog from 'posthog-js'
+import { PostHogProvider } from 'posthog-js/react'
+
+if (typeof window !== 'undefined') {
+  posthog.init(process.env.NEXT_PUBLIC_POSTHOG_KEY, {
+    api_host: process.env.NEXT_PUBLIC_POSTHOG_HOST
+  })
 }
```

Comment:
As this is structured now, we have:
1. Intro
2. Pageview
3. Provider
4. Text
5. Layout

It is a bit jarring to have the provider and pageview back to back. Could we do the simpler provider first, then have some text talking about the pageview component and what it does. 

At the very least, have the provider first.